cnd/src/main.rs

```diff
@@ -127,7 +127,7 @@ fn main() -> anyhow::Result<()> {
     let swap_communication_states = Arc::new(SwapCommunicationStates::default());
 
     // HALight
-    let invoice_states = Arc::new(States::default());
+    let halight_state = Arc::new(States::default());
```

Comment:
```suggestion
    let halight_states = Arc::new(States::default());
```

Reply:
in accordance with all the other state names I made it singular.
(alpha_ledger_state, beta_ledger_state)

is this semantically different in lightning, so plural would be justified?

Reply:
It is not different but I guess it is more the other states that should be renamed :)

Reply:
In Rust, the convention is to have the name of the variable matching the type of the variable as closely as possible.

Reply:
OK, then I rename all of them 😄

Reply:
I'll do this rename in a follow-up PR